### PR TITLE
Update Quickstart, AddRemark and Shift Cmd Summary

### PR DESCRIPTION
**Pull request by jieqiboh**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -169,15 +189,27 @@ Examples:
 
 ### Adding a remark: `addremark`
 
-Adds a remark to an existing internship in the application.
+Adds a remark or modifies the existing one, of an to an existing internship at the specified `INDEX`.
 
-Format: `addremark INDEX [/remark REMARK]`
+<span style="color: #f66a0a;">Format:</span> `addremark INDEX [/remark REMARK]`
```
Comment (CaptainKeqing) [inline docs/UserGuide.md:194]: Should the square brackets be remove to indicate non - optional remark prefix

File: docs/UserGuide.md [hunk 2]
```diff
@@ -169,15 +189,27 @@ Examples:
...
+
+💡 **Try It Out:**<br>
+
+1. Input:<br>
+   `addremark 1 /remark This internship has a behavioural interview!`<br>
+   ![Before adding a remark](./images/add-remark/addremark-before.png)<br><br>
+
+2. Successful Output after adding a remark:<br>
+   ![After successfully adding a remark](./images/add-remark/addremark-after.png)
+</div>
```
Comment (CaptainKeqing) [inline docs/UserGuide.md:212]: Should there be a common errors section as disccused in our meeting? eg blank remark, error output and solution to fix it (not be blank)

File: docs/UserGuide.md [hunk 3]
```diff
@@ -49,18 +49,23 @@ Start keeping track of your internships easily with <span style="color: #f66a0a;
 
 ### Installation
 
-1. Ensure that you have [Java 11 or above](https://www.java.com/en/download/) installed on your computer.
-    - If you are a MacOS, follow the instructions [here](https://nus-cs2103-ay2324s2.github.io/website/admin/programmingLanguages.html) instead.
+1. Ensure that you have **Java 11 or above** installed on your computer.
+    - For MacOS, follow the instructions [here](https://nus-cs2103-ay2324s2.github.io/website/admin/programmingLanguages.html).
+    - For Windows or Linux, follow the instructions [here](https://www.java.com/en/download/). 
 2. Download the latest `CareerSync.jar` file [here](https://github.com/AY2324S2-CS2103T-W11-1/tp/releases/tag/v1.2).
```
Comment (CaptainKeqing) [inline docs/UserGuide.md:55]: Actually navigating to our release page, our jar file is in lowercase. Should this be lowercase too, or the release name changed to CamelCase for consistency

File: docs/UserGuide.md [hunk 4]
```diff
@@ -49,18 +49,23 @@ Start keeping track of your internships easily with <span style="color: #f66a0a;
...
 3. Make sure your jar file is located in an empty folder.
 4. Start <span style="color: #f66a0a;">CareerSync</span>.
-    - For MacOS:
-        - Open up your Terminal by typing <kbd>Command</kbd> + <kbd>Space</kbd>, then type <kbd>Enter</kbd>.<br>
+    - For **MacOS**:
+        - Open up your Terminal by typing <kbd>Command</kbd> + <kbd>Space</kbd>, then type `Terminal` and <kbd>Enter</kbd>.<br>
         - Navigate to the folder containing your jar file using `cd`. If you are not sure how to use `cd`, refer to [this link](https://www.ibm.com/docs/en/aix/7.2?topic=directories-changing-another-directory-cd-command)!<br>
         - Enter `java -jar CareerSync.jar` and type <kbd>Enter</kbd>.
```
Comment (CaptainKeqing) [inline docs/UserGuide.md:61]: Same comment as above on careersync.jar file name, apply to rest of UG too

File: docs/UserGuide.md [hunk 5]
```diff
@@ -169,15 +189,27 @@ Examples:
 
 ### Adding a remark: `addremark`
 
-Adds a remark to an existing internship in the application.
+Adds a remark or modifies the existing one, of an to an existing internship at the specified `INDEX`.
```
Comment (CaptainKeqing) [inline docs/UserGuide.md:192]: Grammar typo here

**Review by CaptainKeqing — CHANGES_REQUESTED** [review]
Some small bugs to fix, lmk once changed ! Otherwise the work looks good

File: docs/UserGuide.md (hunk 3 above)
Comment (jieqiboh) [reply docs/UserGuide.md:55]: CareerSync should be kept it in camel case, the jar file was auto-generated by gradle

File: docs/UserGuide.md (hunk 2 above)
Comment (jieqiboh) [reply docs/UserGuide.md:231]: Oh, I situated them in the format section currently

**Review by HM33-Stu — APPROVED** [review]
All requested changes have been addressed

**Pull request merged**